programs/programs/policyengine/policyengine.py

```diff
@@ -397,4 +398,8 @@ def policy_engine_prepare_params(screen):
         policy_engine_params['household']['households']['household']['members'].append(member_id)
         policy_engine_params['household']['spm_units']['spm_unit']['members'].append(member_id)
 
+    for member_1, member_2 in screen.relationship_map().items():
```

Comment:
Does this assume that the second household member that was entered is the first household member's spouse?

Reply:
No, it assumes the first person with the relationship status of spouse is the spouse, and it also assumes that if there are 2 of `parent`, `fosterParent`, `stepParent`, or `grandParent` that those are spouses as well. You can see how the relationship map works [here](https://github.com/Gary-Community-Ventures/benefits-api/blob/main/screener/models.py#L185).